miner.go

```diff
@@ -156,13 +166,9 @@ func (m *Miner) Start() {
 }
 
 func (m *Miner) initializeFirstRun() error {
-	l := true
-	d := false
-	_, err := m.runtime.rpc.CreateWallet(&gbitcoin.CreateWalletRequest{
-		WalletName:    "default",
-		LoadOnStartup: &l,
-		Descriptors:   &d,
-	})
+	// HACK - for some reason glightning's bitcoind does have
+	// the CreateWallet method, so use the elements client
+	_, err := m.runtime.elem.CreateWallet("default")
```

Comment:
Interesting hack.
There's also the option to get this part merged into elementsproject/glightning: https://github.com/breez/glightning/blob/lspd-working-branch/gbitcoin/bitcoind.go#L614-L638

Reply:
Thanks, and for sure since we're forking this is the correct way.